DataDiscrepancyUtil/dataDiffCheck.py

```diff
@@ -1,61 +1,60 @@
+"""
+    Usage:
+    dataDiffCheck.py <old_data_path> <new_data_path>
+
+    Arguments:
+        old_data_path: File path of old building json
+        new_data_path: File path of new building json
+"""
+# old.json being the data from https://api.oregonstate.edu/v1/locations?page[size]=10000&type=building
+# new.json being the data from https://localhost:8082/api/v0/locations?page[size]=10000&type=building - locations-frontend-api commit b1bec1e013cf29f16a0a01b9dd7c3777e3b4e192
+
+from docopt import docopt
+print(docopt(__doc__, version='1.0.0rc2'))
+
+args = docopt(__doc__, version='1.0.0rc2')
+
 import sys
 import json
 
+def create_mappings(buildings_json):
+    building_data = {}
+    names2ids = {}
+    #map id to building data
+    #map name to id
+    for building in buildings_json['data']:
+        building_data[building['id']] = building
+        names2ids[ building['attributes']['name'] ] = building['id']
+        
+    return (building_data, names2ids)
+
 if __name__ == "__main__":
-    print("Expecting old.json and new.json as input files 1 and 2.")
-    # old.json being the data from https://api.oregonstate.edu/v1/locations?page[size]=10000&type=building
-    # new.json being the data from https://localhost:8082/api/v0/locations?page[size]=10000&type=building - locations-frontend-api commit b1bec1e013cf29f16a0a01b9dd7c3777e3b4e192
-
-    if(len(sys.argv) >= 3):
-        if(sys.argv[1] == sys.argv[2]):
-            print "These are the same file..."
-        print sys.argv[1] + " as old_file"
-        print sys.argv[2] + " as new_file"
-
-        with open(sys.argv[1], "r") as old_file:
-            old_building_data_json = json.loads(old_file.read())
-        with open(sys.argv[2], "r") as new_file:
-            new_building_data_json = json.loads(new_file.read())
-
-        new_building_data_dict = {}
-        old_building_data_dict = {}
-
-        # TODO Rename this name view dict
-        # This is for the NAME : {OLD, NEW} stuff
-        new_bdata_name_dict = {}
-        old_bdata_name_dict = {}
-
-        # Create a dict of the new building data to make the next loop simpler
-        for new_building in new_building_data_json['data']:
-            new_building_data_dict[new_building['id']] = new_building
-            new_bdata_name_dict[new_building['attributes']
-                                ['name']] = new_building['id']
-
-        for old_building in old_building_data_json['data']:
-            old_building_data_dict[old_building['id']] = old_building
-            old_bdata_name_dict[old_building['attributes']
-                                ['name']] = old_building['id']
+    
+    if(args['<old_data_path>'] == args['<new_data_path>']):
+        print "These are the same file..."
+        sys.exit(1)
+    else:
+
+        with open(args['<old_data_path>'], "r") as old_json_file:
+            with open(args['<new_data_path>'], "r") as new_json_file:
```

Comment:
You could open multiple files like this:

```python
with open('a', 'w') as a, open('b', 'w') as b:
    do_something()
```

https://stackoverflow.com/questions/4617034/how-can-i-open-multiple-files-using-with-open-in-python